ccm_web/client/src/App.tsx

```diff
@@ -58,7 +63,10 @@ function App (): JSX.Element {
   if (!globals.user.hasCanvasToken) {
     return (
       <div className='App'>
-        <Breadcrumbs />
+        <div className={classes.breadcrumbsHelpContainer}>
+          <Breadcrumbs />
+          <Help pathnames={[]} features={features} baseHelpURL={globals.baseHelpURL} />
```

Comment:
If it stays like this, we could just make `pathnames` and `features` props optional, you don't need them in this case, since you're just using the base URL.